### PR TITLE
WIP: support for ORM + ODM loaders at the same time DO NOT MERGE

### PR DESCRIPTION
**Pull request by dkarlovi**

### CODE REVIEW

File: resources/config/command.xml [hunk 1]
```diff
@@ -13,6 +13,19 @@
 
             <tag name="console.command" />
         </service>
+
+        <service id="hautelook_alice.console.command.doctrine.doctrine_load_data_fixtures_command"
+                 class="Hautelook\AliceBundle\Console\Command\Doctrine\DoctrineLoadDataFixturesCommand"
+                 public="true">
+            <argument type="string">hautelook:fixtures:load-all</argument>
```
[the remark this answers is not in the document]
Comment (dkarlovi) [reply resources/config/command.xml]: Not sure if we can keep it be 100% compatible, this is the first draft only.

File: src/Loader/DoctrineLoader.php [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+use Hautelook\AliceBundle\DoctrineLoaderInterface as AliceBundleLoaderInterface;
+use Hautelook\AliceBundle\LoggerAwareInterface;
+use InvalidArgumentException;
+use LogicException;
+use Nelmio\Alice\IsAServiceTrait;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+final class DoctrineLoader implements AliceBundleLoaderInterface, LoggerAwareInterface
```
[the remark this answers is not in the document]
Comment (dkarlovi) [reply src/Loader/DoctrineLoader.php]: Not sure what you mean here.

**Comment by dkarlovi** [comment]
@theofidry it shouldn't conflict with https://github.com/theofidry/AliceDataFixtures/pull/99, this would only collect all available `ObjectMnager`s from all registries (ORM, ODM) in a single array and pass it there.

**Comment by dkarlovi** [comment]
Requires https://github.com/theofidry/AliceDataFixtures/pull/107

**Comment by dkarlovi** [comment]
BTW this is functional as is, I'm able to load fixtures to both ORM and ODM at the same time, with ODM having references to ORM using `@notation`, as usual.

**Comment by dkarlovi** [comment]
> BTW this is functional as is

Spoke too soon, this is **mostly** functional, but still needs to purge on all managers too.

**Comment by shehi** [comment]
Really need to be able to load fixtures both on ORM and ODM at the same time. Any idea when this work will be finalized?

**Comment by dkarlovi** [comment]
@shehi I have since moved on to other projects which don't use AliceBundle currently, if you require this merged, I'd recommend continuing the PR, hopefully @theofidry will have some time to review and merge. :pray:

**Comment by data219** [comment]
any chance this gets finalized? if it is just thoose 2 tiny comments that should be taken care of before merge it could gladly takeover if the original author is no longer on bord....

**Comment by dkarlovi** [comment]
@data219 feel free to take over, please. :1st_place_medal:

**Comment by data219** [comment]
Nearly missed the notification about your answer =) Let me see if I can push it the last few meters over the finish line... @theofidry anything else what keeps you from merging besides what you already commented?

**Comment by theofidry** [comment]
@data219 will try to have a look tonight

**Comment by smilesrg** [comment]
@dkarlovi @theofidry any news? :-)

**Comment by dkarlovi** [comment]
I'm not working on this anymore, sorry.

**Pull request closed without merging**